app/src/main/java/com/ahmettekin/WeatherApp/view/ListFragment.java

```diff
@@ -102,4 +149,29 @@ public void onDestroy() {
         super.onDestroy();
         compositeDisposable.clear();
     }
+
+    private void removeFromSuperView(ViewGroup childView) {
```

Comment:
Bu fonksiyonu StringOperations gibi bir class oluşturup onun içine mi alsak?

> ViewHelper

> ViewUtils

> ViewOperations

aklıma bu isimler geldi class ismi olarak.Bunlardan biri bence uygun ya da senin aklına gelen varsa onu kullanabilirsin 👍

Reply:
Tabiki. Yorumların için teşekkür ederim :) 
Hallediyorum 👍